vnext/Microsoft.ReactNative.Cxx/JSI/JsiApiContext.h

```diff
@@ -16,7 +17,7 @@ namespace winrt::Microsoft::ReactNative {
 // Get JSI Runtime from the current JS dispatcher thread.
 // If it is not found, then create it and store it in the context.Properties().
 // Make sure that the JSI runtime holder is removed when the instance is unloaded.
-facebook::jsi::Runtime &GetOrCreateContextRuntime(ReactContext const &context) noexcept {
+inline facebook::jsi::Runtime &GetOrCreateContextRuntime(ReactContext const &context) noexcept {
```

Comment:
I plan to do it in another PR. My initial attempt to do it failed because this method brings new dependencies. Some of our unit test projects require changes to support this method in a .cpp file. I would like not to extend the scope of this PR.

Reply:
I have moved this method implementation to a .cpp file.